src/lib.rs

```diff
@@ -1,14 +1,48 @@
 use std::fmt;
 
-/// TODO...
+/// A trait that can be used for converting between different color models
+/// and performing various transformations on them.
 pub trait Color {
-    // TODO...
+    /// Constructs a valid CSS color string for a given color type.
```

Comment:
I guess for consistency... "Converts `self` to its CSS string format." (note the backpack around `self`)